library/general/src/lib/y2user/configuration.rb

```diff
@@ -0,0 +1,52 @@
+module Y2User
```

Comment:
I am not sure about using the same namespace in yast2 and yast-users-ng. I was discussing it with @ancorgs yesterday. I tend to see yast2 as a generic library with shared functionality. But using the same namespace as other modules looks like a collection of parts of modules that were moved for convenience.

Currently we have very varied namespaces in yast2: `Yast::Execute`, `Yast2::Popup`, `CWM::Table`. IMHO, we should decide about the namespace rules of code provided by yast2. For example, I like what we did for systemd classes, that is, to use `Yast2` as outer namespace and then creating inner namespaces if needed: `Yast2::CompoundAction`, `Yast2::Systemd::Service`, `Yast2::Systemd::Unit`, etc.

For the users classes, maybe we could have something like: `Yast2::Users::Configuration`, `Yast2::Users::User`, `Yast2::Users::Group`, etc. And yast2-users-ng can use `Yast2::Users` classes directly or provide new classes based on the yast2 ones (e.g., `Y2Users::User`) if needed.

What do you think?

Reply:
to be honest I do not have strong opinion here, but what really puzzle me is that inconsistency. So we should not change it when we decide.

Reply:
btw see https://github.com/yast/yast-yast2/pull/1151#discussion_r610461401

Reply:
Yes, if fact that comment was what ring the bell. Having different modules populating the same namespace is something I don't like. I see it as kind of monkey patching. Anyway, if we have already decided to go with that approach, fine with me. But if there is no a clear criteria, then I would suggest to open the discussion the sooner the better.